metadata.rb

```diff
@@ -57,22 +57,19 @@
         type: "object",
         title: "Provider",
         title_es: "Proveedor",
-        required: ["provider", "country"],
-        order: ["provider","country"],
+        required: ["country","provider"],
+        order: ["country","provider"],
         properties: { 
           provider: {
             type: "string",
             title: "Provider",
             title_es: "Proveedor",
```

Comment:
"Provider" will not be selectable in the policy page?

Reply:
Yes, it is. But country field is selectabled firstly to show country's operators.